config/patches/openresty/v1.11.2.1_bundled-nginx-cve-2017-7529.patch

```diff
@@ -0,0 +1,73 @@
+diff -ru openresty-1.11.2.1/bundle/nginx-1.11.2/docs/html/index.html openresty-1.11.2.4/bundle/nginx-1.11.2/docs/html/index.html
+--- openresty-1.11.2.1/bundle/nginx-1.11.2/docs/html/index.html	2016-08-25 02:20:57.000000000 +0200
++++ openresty-1.11.2.4/bundle/nginx-1.11.2/docs/html/index.html	2017-07-11 19:36:36.000000000 +0200
+@@ -16,7 +16,7 @@
+ working. Further configuration is required.</p>
+
+ <p>For online documentation and support please refer to
+-<a href="https://openresty.org/">openresty.org</a>.<br/>
++<a href="https://openresty.org/">openresty.org</a>.<br/></p>
+
+ <p><em>Thank you for flying OpenResty.</em></p>
+ </body>
+diff -ru openresty-1.11.2.1/bundle/nginx-1.11.2/src/core/nginx.h openresty-1.11.2.4/bundle/nginx-1.11.2/src/core/nginx.h
+--- openresty-1.11.2.1/bundle/nginx-1.11.2/src/core/nginx.h	2016-08-25 02:20:57.000000000 +0200
++++ openresty-1.11.2.4/bundle/nginx-1.11.2/src/core/nginx.h	2017-07-11 19:36:35.000000000 +0200
+@@ -11,7 +11,7 @@
+
+ #define nginx_version      1011002
+ #define NGINX_VERSION      "1.11.2"
+-#define NGINX_VER          "openresty/" NGINX_VERSION ".1"
++#define NGINX_VER          "openresty/" NGINX_VERSION ".4"
+
+ #ifdef NGX_BUILD
+ #define NGINX_VER_BUILD    NGINX_VER " (" NGX_BUILD ")"
+diff -ru openresty-1.11.2.1/bundle/nginx-1.11.2/src/core/ngx_resolver.c openresty-1.11.2.4/bundle/nginx-1.11.2/src/core/ngx_resolver.c
+--- openresty-1.11.2.1/bundle/nginx-1.11.2/src/core/ngx_resolver.c	2016-07-18 04:19:50.000000000 +0200
++++ openresty-1.11.2.4/bundle/nginx-1.11.2/src/core/ngx_resolver.c	2017-07-11 19:36:36.000000000 +0200
```

Comment:
This doesn't look strictly necessary for the security fix.  If I'm understanding the surrounding code correctly this is ensuring that nginx errors out if IPv6 was requested but isn't supported in the build.